starters/deno-oak-denodb/src/db/resolvers/technology.ts

```diff
@@ -0,0 +1,51 @@
+import { Technologies } from '../model/technology.ts';
+
+export interface ITechnology {
+	displayName: string;
+	description: string;
+	url: string;
+}
+
+export const technologyResolvers = {
+	Query: {
```

Comment:
Do you think we could put query handlers to a separate module, as well as mutations to its own file?

Reply:
I have moved it, but I am not sure if that's what you mean